ros2controlcli/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ros2controlcli</name>
+  <version>0.0.0</version>
```

Comment:
Maybe to change version number from zero? Or do we want to do this on repository level before release?

Reply:
the version gets bumped automatically the moment you release the package